www/api/class/centreon_ldap_synchro.class.php

```diff
@@ -78,16 +78,12 @@ public function postRequestLdapSynchro(): bool
             $_POST['contactId'] ?? null,
             FILTER_VALIDATE_INT
         );
-        $sessionId = filter_var(
-            $_POST['sessionId'] ?? null,
-            FILTER_SANITIZE_STRING
-        );
 
         if (!$this->isLdapEnabled()) {
             return $result;
         }
 
-        if (empty($contactId) && empty($sessionId)) {
+        if (empty($contactId)) {
```

Comment:
You can remove the check on line 97 too :
https://github.com/centreon/centreon/pull/8876/commits/238686844d5e4385eaa05de87382aeb25202704f#diff-d7e73c17e00be39d953466d59c18aaa5R97